cmd/vcfFilter/testdata/expected_weakToStrongOnly.vcf

```diff
@@ -0,0 +1,4 @@
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	First	Second
```

Comment:
I see this vcf file and the others always listing the ancestral allele (AA) as T, even when T is not a ref or alt allele.

Reply:
This is a copy paste error I can fix on the next commit.